openapi/publicapi_spec.json

```diff
@@ -1221,7 +1440,7 @@
           "authorisation_mode" : {
             "type" : "string",
             "description" : "How the payment will be authorised. Payments created in `web` mode require the paying user to visit the `next_url` to complete the payment.",
-            "enum" : [ "web", "moto_api", "external" ]
+            "enum" : [ "web", "moto_api", "agreement", "external" ]
           },
```

Comment:
Do we want to publish this change? Are we still hiding stuff about agreements?

Reply:
:face_palm

Reply:
👍